.circleci/config.yml

```diff
@@ -14,6 +14,19 @@ jobs:
     working_directory: /go/src/github.com/bbriggs/bitbot
     steps:
      - checkout
+     - run:
+        name: Get needed dependencies
+        command: |
+            go get "github.com/bbriggs/quotes/model"
```

Comment:
We're using go modules, so make sure that Circle is using the correct version of Go (>=1.12) and then you should be able to `go run` and go will take care of deps on its own.